modules/carousel/jetpack-carousel.php

```diff
@@ -701,23 +701,23 @@ function carousel_display_exif_sanitize( $value ) {
 	}
 
 	function carousel_display_geo_callback() {
-		$this->settings_checkbox( 'carousel_display_geo', __( 'Show map of photo location in carousel, when available.', 'jetpack' ) );
+		$this->settings_checkbox( 'carousel_display_geo', esc_html__( 'Show map of photo location in carousel, when available.', 'jetpack' ) );
```

Comment:
If escaped at all, this is best escaped in the `settings_checkbox` method, this will be closer to output and help keep code DRY.